api/etcd/rulesets_test.go

```diff
@@ -24,7 +25,7 @@ var (
 	endpoints   = []string{"localhost:2379", "etcd:2379"}
 )
 
-func Init() {
+func init() {
```

Comment:
Do you mean is it worth having an init method? or the `rand.Seed` call?

Reply:
the rand.Seed call.

Reply:
Yes because in case a test fails and doesn't properly clean up the database, we might open the same namespace and have duplicate errors. That's actually what was happening  before I realized `init` was spelled wrongly

Reply:
oooh that's bad, and really should be dealt with by a helper IMHO. I'll just mention github.com/juju/postgrestest one more time :) :)

Reply:
I've moved it to the test helper